tasks/main.yml

```diff
@@ -22,14 +23,15 @@
   block:
     - set_fact:
         create_db_name: "{{ namespace | truncate(50, true, '') }}_{{ lookup('password', '/tmp/_name length=5 chars=ascii_letters') }}"
-        create_db_user: "{{ lookup('password', '/tmp/_user length=16 chars=ascii_letters') }}"
+        create_db_user: "{{ namespace | truncate(50, true, '') }}_{{ lookup('password', '/tmp/_user length=16 chars=ascii_letters') }}"
```

Comment:
this is gonna fail very very very hard: https://dba.stackexchange.com/questions/147857/username-length-restriction-in-different-mysql-environments 
to be safe we should have mysql usernames max 16 chars (or maybe 32 characters, if we have mysql version `5.7.8` everywhere)

Reply:
ah thanks @Schnitzel. I only checked (and tested against) [mariadb](https://mariadb.com/kb/en/library/identifier-names/#maximum-length), which has a limit of 80.

Reply:
RDS still support mysql 5.5(!), so a limit of 16 seems prudent.